assets/dev/js/editor/regions/responsive-bar/index.js

```diff
@@ -0,0 +1,16 @@
+import ResponsiveBar from 'elementor-regions/responsive-bar/responsive-bar';
+
+elementor.on( 'preview:loaded', ( isFirstLoad ) => {
```

Comment:
Could it be a timing issue? When is the `preview:loaded` triggered before the script is loaded?